[2024-09-04T19:20:07.698+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-09-04T19:20:07.709+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: user_input_2sum.get_sum manual__2024-09-04T19:19:21+00:00 [queued]>
[2024-09-04T19:20:07.714+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: user_input_2sum.get_sum manual__2024-09-04T19:19:21+00:00 [queued]>
[2024-09-04T19:20:07.716+0000] {taskinstance.py:2856} INFO - Starting attempt 2 of 2
[2024-09-04T19:20:07.736+0000] {taskinstance.py:2879} INFO - Executing <Task(PythonOperator): get_sum> on 2024-09-04 19:19:21+00:00
[2024-09-04T19:20:07.745+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=1148) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-09-04T19:20:07.745+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'user_input_2sum', 'get_sum', 'manual__2024-09-04T19:19:21+00:00', '--job-id', '120', '--raw', '--subdir', 'DAGS_FOLDER/python_operator_dag_with_userinput.py', '--cfg-path', '/tmp/tmpbvr172rg']
[2024-09-04T19:20:07.747+0000] {standard_task_runner.py:72} INFO - Started process 1149 to run task
[2024-09-04T19:20:07.748+0000] {standard_task_runner.py:105} INFO - Job 120: Subtask get_sum
[2024-09-04T19:20:07.805+0000] {task_command.py:467} INFO - Running <TaskInstance: user_input_2sum.get_sum manual__2024-09-04T19:19:21+00:00 [running]> on host 626f2e28dec2
[2024-09-04T19:20:07.864+0000] {taskinstance.py:3122} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='abhay' AIRFLOW_CTX_DAG_ID='user_input_2sum' AIRFLOW_CTX_TASK_ID='get_sum' AIRFLOW_CTX_EXECUTION_DATE='2024-09-04T19:19:21+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-09-04T19:19:21+00:00'
[2024-09-04T19:20:07.866+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-09-04T19:20:07.877+0000] {logging_mixin.py:190} INFO - Received num1: 10, num2: 24
[2024-09-04T19:20:07.878+0000] {logging_mixin.py:190} INFO - Sum: 34
[2024-09-04T19:20:07.879+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-09-04T19:20:07.886+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-09-04T19:20:07.887+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=user_input_2sum, task_id=get_sum, run_id=manual__2024-09-04T19:19:21+00:00, execution_date=20240904T191921, start_date=20240904T192007, end_date=20240904T192007
[2024-09-04T19:20:07.925+0000] {local_task_job_runner.py:261} INFO - Task exited with return code 0
[2024-09-04T19:20:07.935+0000] {taskinstance.py:3891} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-09-04T19:20:07.937+0000] {local_task_job_runner.py:240} INFO - ::endgroup::
